docs/how-to/publish-slice.rst

```diff
@@ -0,0 +1,23 @@
+How to make custom slice definitions available for everyone?
+************************************************************
+
+At this stage, you have created some package slice definitions and you have a custom Chisel release in your local development environment.
+You have also tested this custom Chisel release, and it works! You believe there are others who could really use it as well, so **how can you make it accessible to everyone**?
+
+It is as simple as proposing your changes into the upstream `Chisel releases repository <https://github.com/canonical/chisel-releases>`_:
+
+Create a fork of this repository: https://github.com/canonical/chisel-releases::
```

Comment:
can you please bring the bullet list from before? you have nested ":" in this line now